views/js/pciCreator/dev/audioRecordingInteraction/runtime/scss/audioRecordingInteraction.scss

```diff
@@ -181,6 +182,95 @@ $playback: whiten($info, .3);
 
     }
 
+    // Countdown timer as a pie chart
+    $sizeCountdown: 48px;
```

Comment:
We can see the bar behind the pie, it is possible to increase the pie size, or other solution?